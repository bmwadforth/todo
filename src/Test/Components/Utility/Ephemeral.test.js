import React from 'react';
import {shallow, configure} from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import Ephemeral from "../../../Components/Utility/Ephemeral";
configure({adapter: new Adapter()});

describe("Ephemeral Component", () => {
    it("Should Match Snapshot", () => {
        const wrapper = shallow(<Ephemeral countdown={1000}><p>Test</p></Ephemeral>);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
