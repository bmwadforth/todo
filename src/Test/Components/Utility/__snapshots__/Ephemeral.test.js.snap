// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ephemeral Component Should Match Snapshot 1`] = `"<p>Test</p>"`;
